app/helpers/meta_tags_helper.rb

```diff
@@ -0,0 +1,178 @@
+module MetaTagsHelper
+  def meta_title
+    if current_page?(products_development_metrics_path)
+      product_name.present? ? "#{product_name} summary" : default_description
+    elsif current_page?(products_metrics_development_metrics_path)
+      product_name + ' - ' + MetricDefinition.where(code: @query_metric_name).first&.name
+    else
+      title_for_pages
+    end
+  end
+
+  def meta_description
+    if current_page?(products_development_metrics_path)
+      product_name.present? ? product_summary_description : default_description
+    elsif current_page?(products_metrics_development_metrics_path)
+      product_description
+    elsif current_page?(repositories_development_metrics_path)
+      "#{@repository[:name]} summary"
+    else
+      description_for_pages
+    end
+  end
+
+  def meta_image
+    generate_url_image if current_page?(products_metrics_development_metrics_path)
+  end
+
+  private
+
+  def title_for_pages
+    if current_page?(repositories_development_metrics_path)
+      "#{@repository[:name]} summary"
+    elsif current_page?(open_source_index_path)
+      'Open Source repositories'
```

Comment:
What do you think about using `I18n` for plain text occurrences such as titles and descriptions?

Reply:
Done!